pkg/apis/certificates/validation/validation_test.go

```diff
@@ -261,7 +263,7 @@ func TestValidateCertificateSigningRequest(t *testing.T) {
 	}
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
-			el := ValidateCertificateSigningRequest(&test.csr)
+			el := ValidateCertificateSigningRequest(&test.csr, test.gv)
```

Comment:
(as a marker/reminder for me for later): needs more tests 😅

Reply:
so many tests added